res/translations/es.json

```diff
@@ -0,0 +1,230 @@
+    "ReplayGain mode": "Modo ReplayGain",
+    "ReplayGain preamp": "Preamp ReplayGain",
+    "Restart required": "Reinicio requerido",
+    "Save play queue on exit": "Guardar cola de reproducción al salir",
+    "Saved at": "Guardado en",
+    "Scrobble when": "Scrobble cuando",
+    "Search Everywhere": "Buscar en todas partes",
+    "Search page": "Página de búsqueda",
+    "Search playlists or new playlist name": "Buscar listas de reproducción o nombre de nueva lista",
+    "sec": "seg",
+    "selected": "seleccionado",
+    "Send playback statistics to server": "Enviar estadísticas de reproducción al servidor",
+    "Server": "Servidor",
+    "Server Type": "Tipo de servidor",
+    "Server unreachable": "Servidor inalcanzable",
+    "Set rating": "Establecer calificación",
+    "Settings": "Configuración",
+    "Share": "Compartir",
+    "Share content": "Compartir contenido",
+    "Show": "Mostrar",
+    "Show info": "Mostrar información",
+    "Show notification on track change": "Mostrar notificación en cambio de pista",
+    "Show play queue": "Mostrar cola de reproducción",
+    "Show year in album grid cards": "Mostrar año en tarjetas de cuadrícula de álbumes",
+    "Shuffle": "Aleatorio",
+    "Shuffle albums": "Aleatorio álbumes",
+    "Shuffle tracks": "Aleatorio pistas",
+    "Similar artists": "Artistas similares",
+    "Single": "Sencillo",
+    "Size": "Tamaño",
+    "Skip duplicate tracks": "Saltar pistas duplicadas",
+    "Skip this version": "Saltar esta versión",
+    "Soundtrack": "Banda sonora",
+    "Spoken Word": "Palabra Hablada",
+    "Startup page": "Página de inicio",
+    "Stopped": "Detenida",
+    "Support the project": "Apoya el proyecto",
+    "Switch Servers": "Cambiar servidores",
+    "Testing connection": "Probando conexión",
+    "Theme": "Tema",
+    "Time": "Tiempo",
+    "Title": "Título",
+    "Title (A-Z)": "Título (A-Z)",
+    "Top Tracks": "Mejores Pistas",
+    "Total time": "Tiempo total",
+    "Track": "Pista",
+    "track": "pista",
+    "Track count": "Número de pistas",
+    "Track gain": "Ganancia de la pista",
+    "Track Info": "Información de la pista",
+    "Track number": "Número de pista",
+    "Track peak": "Pico de la pista",
+    "tracks": "pistas",
+    "UI Scaling": "Escalado de la interfaz",
+    "URL": "URL",
+    "Use legacy authentication": "Usar autenticación legacy",
+    "Username": "Nombre de usuario",
+    "version": "versión",
+    "Visualizations": "Visualizaciones",
+    "Volume": "Volumen",
+    "wrong URL": "URL incorrecta",
+    "wrong username/password": "nombre de usuario/contraseña incorrectos",
+    "Year": "Año",
+    "Year (ascending)": "Año (ascendente)",
+    "Year (descending)": "Año (descendente)",
+    "Year from": "Año desde",
+    "You are running the latest version of": "Estás ejecutando la última versión de",
+    "Quit": "Salir",
+    "Rescan Library": "Volver a escanear la biblioteca",
```

Comment:
This could probably just be "Escanear la biblioteca" to take up less width in the settings menu - what do you think?

Reply:
I initialy wasn't sure `reescanear` was gramatically correct but seems it should be right according to this article: https://electivolit.blogspot.com/2019/09/todo-sobre-los-prefijos-re-escanear-o.html